fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
Yes, that naming is much better.

During my traverse for `displayName`, I am also going to have a look at what sort of things exist inside the kickers when they are `HTML`. Maybe we will be able to keep `HTML` out of it, but i'm not sure as I am not very familiar with `Kickers`, this is very much a `frontend` concept. Hopefully we can avoid any nasty `HTML` stripping.

I can't see what the problem is with resolving this text yourself though, I mean, you have the `types`! :smile: The only benefit is if we did start using it you would have consistency across app and web, but are we looking to have this? Design in the apps can be different to the design on the web!